### DISCUSSION TITLE
[HackGPT] Bad decryption of encrypted message

### DISCUSSION THREAD

**Discussion by 727yubin** [prosyslab-classroom/cs348-information-security#757] [HackGPT]
![Screenshot from 2023-03-06 11-18-52](https://user-images.githubusercontent.com/14364385/223004663-1ff722ad-0523-4f30-baac-27566f5af5d1.png)

## Description

I don't know if this is HackGPT-worthy, nevertheless it seems interesting.

After the lecture today, I tried decrypting the message #209. It failed, but in a quite interesting way?
It seems that ChatGPT recognizes the strings "noM" and ".lla iH", and tries to fit it in with something that it has seen.

**Comment by KihongHeo** [comment]
Yubin, you indeed tried that. Interesting output.

But, this means that even such a naive encryption scheme is not fully breakable by ChatGPT. So GPT does not become a threat in this case (i.e., human wins). Can you make any harmful effects using this result? Then it would be very interesting.
